core/templates/pages/story-viewer-page/story-viewer-page-root.component.ts

```diff
@@ -16,11 +16,12 @@
  * @fileoverview Root component for Story Viewer Page.
  */
 
-import { Component } from '@angular/core';
+import { Component, ViewEncapsulation } from '@angular/core';
 
 @Component({
   selector: 'oppia-story-viewer-page-root',
-  templateUrl: './story-viewer-page-root.component.html'
+  templateUrl: './story-viewer-page-root.component.html',
+  encapsulation: ViewEncapsulation.None
```

Comment:
It seems this is only used in one other page on Oppia. Any guidance for when to use it and when not to?

Reply:
These styles were placed in `story-viewer-page.mainpage.html`. This means these styles are applied to only that page. Putting the styles to `oppia.css` might conflict with other pages. So, I put these styles to a component that is only loaded in the story viewer page, and disabling view encapsulation will also apply styles to other components on this page (which was the original intent).

Reply:
I see, thanks. Just to check ... should we be using this as a standard methodology for all our pages? Otherwise, we'll need something on the wiki to explain to devs when to use it and when not to.

Reply:
Situation 1:
Styles are only applied to one component:
In this case, the styles should be added to the component's template. 

Situation 2:
Styles are shared across components:
In this case, the styles should be placed in `oppia.css`. 

**Situation 3:**
Styles contain some classes which behave differently on a certain page.

example .navbar-brand, .float-left classes in the screenshot below.

![Screenshot from 2021-09-23 12-17-28](https://user-images.githubusercontent.com/24855641/134465034-e4c3f0df-387a-48f9-bf21-378083abedb5.png)

For this case earlier, the styles were placed in `*mainpage.html` files but now they need to be added this way.

Reply:
I see, OK. Would it be possible to create a wiki page along the lines of "How to handle CSS" for this, and explain the rules?

Reply:
(Also /cc @U8NWXD who is looking after the wiki and can advise on where to put this. /cc @DubeySandeep too.)

Reply:
@ashutoshc8101 I think you can add the new page to the sidebar under Developer Reference > Frontend